authenticators/cf_authenticator_test.go

```diff
@@ -155,6 +155,17 @@ var _ = Describe("CFAuthenticator", func() {
 			Expect(metadata).To(Equal(metadata))
 		})
 
+		It("logs the access to the container by the user", func() {
+			Eventually(logger.LogMessages).Should(ContainElement("test.cf-authenticate.app-access-success"))
```

Comment:
Instead of checking logger.LogMessages, usually we use the `gbytes` package in Gomega to verify that the log stream says what we want it to say. For example:

```
Eventually(logger).Should(gbytes.Say("test.cf-authenticate.app-access-success: {\"app\": \"that-app-guid-you-had\", ...}"))
```
I didn't copy the exact log line you need to search for, but I think you get the idea. This means you don't have to do that for loop below either and the test assertion becomes a single line.